akka-actor-tests/src/test/java/akka/actor/JavaAPI.java

```diff
@@ -77,9 +69,13 @@ public Object create() {
     });
   }
 
+  public static Props mkPropsWithLambda() {
+    return Props.create(JavaAPITestActor.class, JavaAPITestActor::new);
```

Comment:
I see no Lambda :)

Reply:
isn't method references just sugar for lambdas?

Reply:
Java pulled the cotton candy over Chris' eyes.